package.json

```diff
@@ -15,7 +15,7 @@
     "babel-runtime": "6.26.0",
     "bluebird": "^3.5.1",
     "case-sensitive-paths-webpack-plugin": "2.1.1",
-    "chalk": "1.1.3",
+    "chalk": "^1.1.3",
```

Comment:
This change is not required.

Reply:
I have taken care of it in current commit.